integration/spark2/src/main/spark2.1/CarbonSessionState.scala

```diff
@@ -133,7 +134,9 @@ class CarbonSessionState(sparkSession: SparkSession) extends HiveSessionState(sp
   override lazy val sqlParser: ParserInterface = new CarbonSparkSqlParser(conf, sparkSession)
 
   experimentalMethods.extraStrategies =
-    Seq(new CarbonLateDecodeStrategy, new DDLStrategy(sparkSession))
+    Seq(new CarbonLateDecodeStrategy,
+      new DDLStrategy(sparkSession)
+    )
```

Comment:
Keep the format as old like
 `Seq(new CarbonLateDecodeStrategy, new DDLStrategy(sparkSession))`

Reply:
Done